src/main/java/UML/views/GraphicalView.java

```diff
@@ -481,29 +481,81 @@ public void deleteClassPanel(String aClass) {
      */
     public void resizePanel(String classToString, int x, int y) {
         JPanel panel = classPanels.get(classToString);
-        Scanner lineScanner = new Scanner(classToString);
-        //Scan the string to get a general idea for its size.
-        int longest = 0;
-        int height = 0;
-        while (lineScanner.hasNextLine()) {
-            String line = lineScanner.nextLine();
+
```

Comment:
I don't think resizePanel is necessary if you never set a preferred size anywhere. There is a way to set a minimum width and minimum height.